components/CodeCard.tsx

```diff
@@ -0,0 +1,26 @@
+"use server";
+
+import { Card, CardContent } from "@/components/ui/card";
+import CopyButton from "./CopyButton";
```

Comment:
Prefer to use @/components/CopyButton for import

Reply:
good catch